### PR TITLE
[SPARK-27151][SQL] ClearCacheCommand extends IgnoreCahedData to avoid plan node copys

### PR DESCRIPTION
**Pull request by maropu**
## What changes were proposed in this pull request?
In SPARK-27011, we introduced `IgnoreCahedData` to avoid plan node copys in `CacheManager`.
Since `ClearCacheCommand` has no argument, it also can extend `IgnoreCahedData`.

## How was this patch tested?
Pass Jenkins.

### CODE REVIEW

File: sql/core/src/main/scala/org/apache/spark/sql/execution/command/cache.scala [hunk 1]
```diff
@@ -83,13 +83,10 @@ case class UncacheTableCommand(
 /**
  * Clear all cached data from the in-memory cache.
  */
-case class ClearCacheCommand() extends RunnableCommand {
+case object ClearCacheCommand extends RunnableCommand with IgnoreCachedData {
```
Comment (dongjoon-hyun) [inline sql/core/src/main/scala/org/apache/spark/sql/execution/command/cache.scala]: Hi, @maropu .
The purpose and title doesn't match. What you want is `IgnoreCahedData` and the test case, isn't it?

Comment (dongjoon-hyun) [reply sql/core/src/main/scala/org/apache/spark/sql/execution/command/cache.scala]: Please update JIRA/PR together.

File: sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala [hunk 2]
```diff
@@ -3013,6 +3013,16 @@ class SQLQuerySuite extends QueryTest with SharedSQLContext {
       sql("reset")
     }
   }
+
+  test("clear cache command should not fail with cache") {
+    withTable("tbl") {
+      val provider = spark.sessionState.conf.defaultDataSourceName
```
Comment (dongjoon-hyun) [inline sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala]: Thank you for using this portable line!

File: sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala [hunk 3]
```diff
@@ -3013,6 +3013,16 @@ class SQLQuerySuite extends QueryTest with SharedSQLContext {
...
+      sql(s"CREATE TABLE tbl(i INT, j STRING) USING $provider")
+      sql("clear cache")
+      sql("cache table tbl")
+      sql("clear cache")
+    }
+  }
```
Comment (dongjoon-hyun) [inline sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala]: This test case doesn't fail without the patch. Please remove this if this is not your intention.

**Review by dongjoon-hyun — CHANGES_REQUESTED** [review]
The test case seems to be not a part of this PR.

File: sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala (hunk 3 above)
Comment (maropu) [reply sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala]: ok

**Comment by maropu** [comment]
@dongjoon-hyun could you check again? Thanks!

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM. Merged to matster. Thank you, @maropu !

**Pull request closed without merging**